src/comps/footer.tsx

```diff
@@ -32,21 +32,17 @@ const contracts = [
   },
 ];
 
-export default function Footer({ config }: { config: BridgeConfig }) {
+export default async function Footer({ config }: { config: BridgeConfig }) {
   const network = getBitcoinNetwork(config.WALLET_NETWORK);
   const { SUPPORT_LINK: supportLink } = config;
-  const { data: aggregateAddress } = useQuery({
-    queryKey: ["aggregateAddress"],
-    queryFn: async () => {
-      const response = await getAggregateKey();
+  const response = await getAggregateKey();
```

Comment:
if this readonly call throws, do we get a "bad" error?

Reply:
good catch this might indeed fail reloading should fix it but it shouldn't happen much since we're depending on hiro infra and this is an internal request

Reply:
fixed